bin/oneoff/data_fix/csedweek_event_to_hoc_signup_2013

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+#
+# This script renames the 'CSEdWeekEven2013' form to 'HocSignup2013'.
+#
+#   UPDATE forms
+#      SET kind = 'HocSignup2013'
+#    WHERE kind = 'CSEdWeekEvent2013';
+#
+
+require File.expand_path('../../../../pegasus/src/env', __FILE__)
+require 'cdo/db'
+require src_dir 'forms'
+
+rows = PEGASUS_DB[:forms].where(kind: 'CSEdWeekEvent2013')
+
+puts "#{__FILE__}: #{rows.sql}"
+STDOUT.flush
+
+rows.each do |row|
+  puts "#{__FILE__}: Fixing ID: #{row[:id]}..."
+  STDOUT.flush
```

Comment:
This could be done via `STDOUT.sync = true` once (rather than needing to `flush` in multiple places)?

Aside, historically our scripts have not bothered to flush to STDOUT, as we have never had issues with the output not appearing.